go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/vmware-tanzu/velero
 
-go 1.22.0
+go 1.22.2
```

Comment:
Shouldn't this be 1.22.3 as we are setting for GCP object store plugin ? https://github.com/vmware-tanzu/velero-plugin-for-gcp/pull/186

Reply:
I recall we reviewed the CVE fix in v1.22.3 and decided to go with v1.22.2 in velero 1.14.0
Let me check with @blackpiglet

Reply:
Clarified with @blackpiglet that we'll use golang v1.22.2